lib/src/Uncertainty/Distribution/ChiSquare.cxx

```diff
@@ -271,7 +271,13 @@ void ChiSquare::computeCovariance() const
 /* Get the moments of the standardized distribution */
 Point ChiSquare::getStandardMoment(const UnsignedInteger n) const
 {
-  return Point(1, std::exp(n * M_LN2 + SpecFunc::LnGamma(n + 0.5 * nu_) - SpecFunc::LnGamma(0.5 * nu_)));
+  return Point(1, std::exp(SpecFunc::LnGamma(n + 0.5 * nu_) - SpecFunc::LnGamma(0.5 * nu_)));
+}
+
+/* Get the standard representative in the parametric family, associated with the standard moments */
+Distribution ChiSquare::getStandardRepresentative() const
+{
+  return Gamma(0.5 * nu_, 1.0, 0.0);
```

Comment:
It is because you want to benefit from the LaguerreFactory specific orthonormal polynomials factories in StandardDistributionPolynomialFactory. The standard representative is an affine transformation of the distribution leading to a distribution with less free parameters and most important a specific polynomial factory if possible.

Reply:
So this means the standard representative of the Chi-square family does not belong to the Chi-square family?

Reply:
Yes. It belongs to the broader family of Gamma distributions. Note that the representative of distributions have been introduced in the unique objective of providing efficient algorithms to build the associated orthonormal polynomials up to an affine transform (which is harmless in the context of PCE). Of course you can use it for other purposes.